### ISSUE TITLE
After new Comfy update yesterday or today. Big problem, all forwarding from Muter and Bypasser no longer works.

### ISSUE THREAD

**Issue by murphylanga** [rgthree/rgthree-comfy#351]
I have uploaded a very large workflow, but it is already at 1 node + mute/bypass repeater + fast muter

[SDXL+Kolors+Flux_BIG_Base_Modul_Styler_V1.01.json](https://github.com/user-attachments/files/17333333/SDXL%2BKolors%2BFlux_BIG_Base_Modul_Styler_V1.01.json)

**Comment by murphylanga** [comment]
It's because of the new frontend from yesterday.
Temporary workaround, go back to an old frontend in the
run_nvidia_gpu.bat with:

--front-end-version Comfy-Org/ComfyUI_legacy_frontend@latest

**Comment by Rekano33** [comment]
Yes there is indeed a problem... Temporary solution helped, I really missed this functionality. Thank you!

**Comment by murphylanga** [comment]
> Yes there is indeed a problem... Temporary solution helped, I really missed this functionality. Thank you!

Yes, but unfortunately the blibla nodes that allow you to pin and unpin all nodes at once don't work with 1.3.6. I'm going to put Comfy in the corner for now. It's not fun at the moment.
